implementations/rust/ockam/ockam_command/tests/commands.bats

```diff
@@ -76,6 +76,28 @@ teardown() {
   rm -rf /tmp/ockam
 }
 
+@test "vault delete" {
+  vault_name=$(openssl rand -hex 4)
+
+  run $OCKAM vault create "${vault_name}"
+  assert_success
+  run $OCKAM vault delete "${vault_name}"
+  assert_success
+  run $OCKAM vault show "${vault_name}"
+  assert_failure
+
+  # Faill to delete vault when it's in use by a node
```

Comment:
I will add another test where it deletes the vault and the identity uses it, along with fixing the typo in the comment